server/src/http/feature_refresher.rs

```diff
@@ -26,6 +28,19 @@ pub struct FeatureRefresher {
     pub persistence: Option<Arc<dyn EdgePersistence>>,
 }
 
+fn client_application_from_token(token: EdgeToken) -> ClientApplication {
+    ClientApplication {
+        app_name: "unleash_edge".into(),
+        connect_via: None,
+        environment: token.environment,
+        instance_id: None,
+        interval: 5,
```

Comment:
Should we include the actual polling interval here? Might be useful

Reply:
Agreed. I'll fix